proposals/0192-non-exhaustive-enums.md

```diff
@@ -61,14 +71,14 @@ To see how this distinction will play out in practice, I investigated the public
 
 In Swift 4.2, enums imported from C and enums defined in the standard library and overlays are either *frozen* or *non-frozen.* (Grammatical note: they are not "unfrozen" because that implies that they were frozen at one point.)
 
-When a client tries to switch over a non-frozen enum, they must include either a `default` case or a new `unknown` case. In Swift 4 mode, omitting this case will result in a warning; in Swift 5 (when there is a Swift 5), it will be an error.
+When a client tries to switch over a non-frozen enum, they should include a "catch-all" case of some kind (`default`, `case _`, etc). In Swift 5 mode, omitting this case will result in a warning.
```

Comment:
Thanks for the update Jordan, but this doesn't make sense to me.  Switches are required to be exhaustive in Swift in general.  A non-frozen enum should require a catch all of some kind, meaning that in Swift 5 this should be an error.

Reply:
I'd love to make it an error in Swift 5 but in practice I think that will just make it harder to migrate to Swift 5. I think this fits our warning policy: "you'd better fix it before production, but it's still reasonable to build and run the program as is". We can totally make it an error in Swift 6, though, when people have broken the habit of writing code the old way.